server/store/user.go

```diff
@@ -51,7 +51,7 @@ func (s *Store) GetUserInfo(userID string) (*UserInfo, error) {
 
 	infoBytes, appErr := s.API.KVGet(tokenKey + userID)
 	if appErr != nil || infoBytes == nil {
-		return nil, errors.New("must connect user account to Microsoft first")
+		return nil, errors.New("connect the user account to Microsoft Teams")
```

Comment:
```suggestion
		return nil, errors.New("your Mattermost account is not connected to your Microsoft Teams account")
```

Reply:
@mickmister Hi, as per your and larkox comment i changed the validation.